lib/components/narrative/narrative-itineraries.js

```diff
@@ -577,7 +580,8 @@ const mapStateToProps = (state) => {
     groupByMode: groupItineraries,
     groupTransitModes,
     mergeItineraries,
-    showHeaderText
+    showHeaderText,
+    sortModes
```

Comment:
Would that be better if `NarrativeItinerariesHeader` became a connected component and derive the props it needs from redux, removing those from `NarrativeItineraries`?

Reply:
Yes, but I don't think that's possible since both of those components get subclassed

Reply:
I think it's a good architecture!